client-participation/js/strings.js

```diff
@@ -56,9 +59,9 @@ preloadHelper.acceptLanguagePromise.then(function() {
     if (languageCode.match(/^en/)) {
       _.extend(strings, translations.en_us);
     }
-    // else if (languageCode.match(/^ja/)) {
-    //   strings = _.extend(strings, translations.ja);
-    // }
+    else if (languageCode.match(/^ja/)) {
+      strings = _.extend(strings, translations.ja);
```

Comment:
@urakagi maybe this?

```suggestion
      _.extend(strings, translations.ja);
```

Reply:
Oh, sure. I'm not sure why there is an assignment.